src/sql/src/plan/statement/ddl.rs

```diff
@@ -1407,7 +1411,217 @@ pub fn plan_create_views(
                 materialize: materialized,
             }))
         }
-        CreateViewsDefinitions::Source { .. } => bail!("cannot create view from source"),
+        CreateViewsDefinitions::Source {
+            name: source_name,
+            targets,
+        } => {
+            let source_connector = normalize::unresolved_object_name(source_name.clone())
+                .map_err(anyhow::Error::new)
+                .and_then(|name| {
+                    scx.catalog
+                        .resolve_item(&name)
+                        .and_then(|item| item.source_connector())
+                        .map(|s| s.clone())
+                        .map_err(anyhow::Error::new)
+                });
+            match source_connector? {
+                SourceConnector::External {
+                    connector:
+                        ExternalSourceConnector::Postgres(PostgresSourceConnector {
+                            conn: _,
+                            publication: _,
+                            details,
+                            ..
```

Comment:
Is having some fields ignored with the `_` pattern and some with the ellipsis on purpose? For me the test is "should this match cause a compile error if the matched struct gets more fields?". It looks like the answer is no here so I'd go for just ellipsis

Reply:
I misunderstood how the operator worked and thought that since we needed a middle field that the leading fields had to be explicitly ignored. 🤦